client/src/components/notifications/MutlipleUsersJoinWorkspaceNotification.vue

```diff
@@ -0,0 +1,99 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <notification-item :notification="notification">
+    <div class="notification-icon">
+      <ion-icon
+        class="file-icon"
+        :icon="people"
+      />
+    </div>
+    <div class="notification-details">
+      <ion-text class="notification-details__message body">
+        <i18n-t
+          keypath="notification.multipleUsersJoinWorkspace"
+          scope="global"
+        >
+          <template #count>
+            <strong> {{ notificationData.roles.length }} {{ $t('notification.users') }} </strong>
+          </template>
+          <template #workspace>
+            <strong>{{ workspaceName }}</strong>
+          </template>
+        </i18n-t>
+      </ion-text>
+      <ion-text class="notification-details__time body-sm">
+        <span
+          class="hover-state"
+          @click="openPopover($event)"
+        >
+          {{ $t('notificationCenter.viewJoinedUsers') }}
+        </span>
+        <span class="default-state">{{ formatTimeSince(notification.time, '', 'short') }}</span>
+      </ion-text>
+    </div>
+  </notification-item>
+</template>
+
+<script setup lang="ts">
+import { formatTimeSince } from '@/common/date';
+import MultipleUsersJoinPopover from '@/components/notifications/MultipleUsersJoinPopover.vue';
+import NotificationItem from '@/components/notifications/NotificationItem.vue';
+import { getWorkspaceName } from '@/parsec';
+import { MultipleUsersJoinWorkspaceData } from '@/services/informationManager';
+import { Notification } from '@/services/notificationManager';
+import { IonIcon, IonText, popoverController } from '@ionic/vue';
+import { people } from 'ionicons/icons';
+import { onMounted, ref } from 'vue';
+
+const workspaceName = ref('');
+const isPopoverOpen = ref(false);
```

Comment:
Is there a point to this variable?

Reply:
To prevent popover opened by error, so it will always be close. But probably unnecessary.